# Defines how fast new measurements are averaged into the transformation estimate (new_value = (1-update_rate)*old_value + update_rate*measurement). [0,1]
# double
update_rate: 0.25

# The name of the computed child frame.
# string
child_frame_name: "/landmark_reference_nav"

# the (minimum) length of the frontal wall segment left of the laser scanner's origin (assuming the robot faces the frontal wall), in [m]
# double, deprecated
# wall_length_left: 2.5

# the (minimum) length of the frontal wall segment right of the laser scanner's origin (assuming the robot faces the frontal wall), in [m]
# double, deprecated
# wall_length_right: 2.5

# height above ground of base frame
# double
base_height: 0.0

# base link of robot
# string
base_frame: "base_link"

# laser scanner topic
# string
laser_scanner_topic_in: "/scan"

# Polygon points that define the area which is used to find the front wall inside.
# Includes x and y coordinates [in m] of each point that define the polygon in the robot's base frame. Input at least 3 points. Do not forget to repeat the first point at the end.
# vector<Point2f>
front_wall_polygon: [0.3, 0.75,
                     3, 0.75,
                     3, -0.75,
                     0.3, -0.75,
                     0.3, 0.75]

# Polygon points that define the area which is used to find the side wall inside.
# Includes x and y coordinates [in m] of each point that define the polygon in the robot's base frame. Input at least 3 points. Do not forget to repeat the first point at the end.
# vector<Point2f>
side_wall_polygon: [0, 3.0,
                    3.0, 3.0,
                    3.0, 0.3,
                    0, 0.3,
                    0, 3.0]
